hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaRolloutExecutor.java

```diff
@@ -0,0 +1,660 @@
+/**
```

Comment:
Did you change the logic in any way or has the code been moved as is?

Reply:
no, everything was just moved as is, reference to the `start` method on the `RolloutManagement` is called via `context.getBean(RolloutManagement.class).start()` see https://github.com/eclipse/hawkbit/blob/4029dbfaa36908ea5a89410e252e89b9bff29f3a/hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaRolloutExecutor.java#L255